src/test/kotlin/org/rust/lang/refactoring/IntroduceVariableTest.kt

```diff
@@ -0,0 +1,60 @@
+package org.rust.lang.refactoring
+
+import org.rust.lang.RustTestCaseBase
+import org.rust.lang.core.psi.RustExprStmtElement
+
+class IntroduceVariableTest : RustTestCaseBase() {
+    override val dataPath = "org/rust/lang/refactoring/fixtures/introduce_variable/"
+
+//    @Ignore
+//    fun testVariable() = checkByFile {
+//        val rustLocalVariableHandler = RustLocalVariableHandler()
+//        openFileInEditor("variable.rs")
+//        rustLocalVariableHandler.invoke(myFixture.project, myFixture.editor, myFixture.file, com.intellij.openapi.actionSystem.DataContext.EMPTY_CONTEXT)
+//    }
+
+
+    fun testMultipleOccurrences() = checkByFile {
```

Comment:
@matklad Oh yes! that looks a lite nicer than the fixture files. I picked those up from the contributing, but this looks a lot nicer